deb/openmediavault/usr/share/php/openmediavault/system/storage/backend/backendabstract.inc

```diff
@@ -134,7 +134,9 @@ abstract class BackendAbstract {
 		foreach(file("/proc/partitions") as $outputk => $outputv) {
 			if (1 !== preg_match($regex, $outputv, $matches))
 				continue;
-			$result[] = sprintf("/dev/%s", $matches[4]);
+			$deviceName = sprintf("/dev/%s", $matches[4]);
+			if (file_exists($deviceName))
+				$result[] = $deviceName;
```

Comment:
Could you please add a note here why this is done. It looks a little bit strange to check for a device that is listed in `/proc/partitions` without having the LXC context here.

Reply:
more info #944

Reply:
Please add a note in the code, too.